src/vue/main-menu.vue

```diff
@@ -1,8 +1,21 @@
 <template>
   <div class="alpheios-alignment-app-menu" id="alpheios-main-menu">
       <div class="alpheios-alignment-app-menu__buttons">
-        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-download" @click="$emit('download-data')" >{{ l10n.getMsgS('MAIN_MENU_DOWNLOAD_TITLE') }}</button>
-        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-upload" @click="uploadTexts" >{{ l10n.getMsgS('MAIN_MENU_UPLOAD_TITLE') }}</button>
+        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-add-target" 
+                @click="$emit('add-target')" :disabled="!docSourceEditAvailable" >
+                {{ l10n.getMsgS('MAIN_MENU_ADD_TARGET_TITLE') }}
+        </button>
+
+        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-download" 
+                @click="$emit('download-data')"  :disabled="!docSourceEditAvailable" >
+                {{ l10n.getMsgS('MAIN_MENU_DOWNLOAD_TITLE') }}
+        </button>
+
+        <button class="alpheios-button-tertiary" id ="alpheios-main-menu-upload" 
+                @click="uploadTexts"  :disabled="!docSourceEditAvailable" >
+                {{ l10n.getMsgS('MAIN_MENU_UPLOAD_TITLE') }}
+        </button>
+

```

Comment:
I think we need to add a handler to catch the selectTab event sent from the align-editor-tabs component so that Undo and Redo can also be disabled when in View Only mode. It doesn't make sense for them to be  active while click is disabled does it?

Reply:
oh, it is an interesting case, will add it